builder/builder-module.c

```diff
@@ -1343,9 +1372,10 @@ builder_module_build (BuilderModule  *self,
         }
     }
 
-  has_configure = g_file_query_exists (configure_file, NULL);
+  if (!simple)
+    has_configure = g_file_query_exists (configure_file, NULL);
 
-  if (!has_configure && !self->no_autogen)
+  if (!simple && !has_configure && !self->no_autogen)
```

Comment:
Rather than a adding !simple && in a lot of place, can't we make a huge if (!simple) and put all the other buildsystems in there.